src/components/ChartContainer/D3Container.js

```diff
@@ -86,7 +90,7 @@ function D3Container({ dashboardState, dashboardActions, store }) {
     if (store.currentResults !== undefined) {
       setSelectedMeasures(store.currentResults.map((result) => result.measure));
     }
-  }, [setSelectedMeasures, setCurrentFilters, store.currentResults]);
+  }, [setSelectedMeasures, setCurrentFilters, setByLineSelectedMeasures, store.currentResults]);
```

Comment:
Does `setByLineSelectedMeasures` need to be added here? It's not being used.

Reply:
I tested it with it removed and just as you thought, it wasn't doing a thing. I'll remove that.